build/l10n-pseudo.js

```diff
@@ -0,0 +1,143 @@
+'use strict';
```

Comment:
Yeah, I can do it here or in a follow-up. @RickyChien, what's your preference? I need to add code for multilocale.js to use when building pseudolanguages.  I can:

1. add it directly in multilocale.js,
2. create a new file and require it from multilocale.js,
3. create `build/l10n/` directory, and put l10n.js, pseudo.js and (possibly) multilocale.js in it.

Zibi's and my preference would be to do #3.  What are your thoughts on this?  Would you rather have us discuss this in a separate bug?  Thanks1